met/docs/Users_Guide/data_io.rst

```diff
@@ -255,24 +255,22 @@ Part of the strength of MET is the leveraging of capability across tools. There
 
 Many of the MET tools use a configuration file to set parameters. This prevents the command line from becoming too long and cumbersome and makes the output easier to duplicate.
 
-Settings common to multiple tools are described in the following sections while those specific to individual tools are explained in the sections for those tools. In addition, these configuration settings are described in the **share/met/config/README** file and the **share/met/config/README-TC** file for the MET-Tropical Cyclone tools.
+Settings common to multiple tools are described in the following sections while those specific to individual tools are explained in the sections for those tools. In addition, these configuration settings are described in the *share/met/config/README* file and the *share/met/config/README-TC* file for the MET-Tropical Cyclone tools.
```

Comment:
I just noticed these references to the config/README files.  I'll plan to fix these up when I modify the test/config files and do it all in that one commit since they are related.